### ISSUE TITLE
create authentication system for administrators

### ISSUE THREAD

**Issue by shivangi316** [CSAllenISD/2023-ISP-ShelterSafe#36]

**Comment by PranavKrishnan05** [comment]
Yeah this is definitely something that will be neccicary once we figure out the user UI and backend

**Comment by omerert** [comment]
If we can finish everything on time, then we can try to implement this

**Comment by kripatailor** [comment]
I agree, we can work on this after we have everything for the app finished and ready to be used.
